test/e2e_test.go

```diff
@@ -144,6 +144,13 @@ func (e *e2e) deployOperator(manifest, profiles string) {
 		manifest,
 	)
 
+	if e.platformSupportsSelinux() {
+		e.run(
+			"sed", "-i", "/EnableSeccomp/a \\ \\ EnableSelinux: \"true\"",
```

Comment:
This `sed` could be simplified once you always add `EnableSelinux` in the sample config.

Reply:
Will do.

Reply:
Done